streaming/src/main/scala/org/apache/spark/streaming/StreamingContext.scala

```diff
@@ -60,7 +60,14 @@ import org.apache.spark.util.{CallSite, ShutdownHookManager, ThreadUtils, Utils}
  * using `context.start()` and `context.stop()`, respectively.
  * `context.awaitTermination()` allows the current thread to wait for the termination
  * of the context by `stop()` or by an exception.
+ * @since 3.4.0
```

Comment:
nit: Remove @since annotation. @since annotation is to represent when this class is introduced. we explicitly say deprecation happens which version in the deprecation message. Apply this to all below.